oqsprov/oqs_sig.c

```diff
@@ -1299,6 +1320,17 @@ static int oqs_sig_set_ctx_params(void *vpoqs_sigctx,
         if (!oqs_sig_setup_md(poqs_sigctx, mdname, mdprops))
             return 0;
     }
+#if (OPENSSL_VERSION_PREREQ(3, 2))
+    p = OSSL_PARAM_locate_const(params, OSSL_SIGNATURE_PARAM_CONTEXT_STRING);
+    if (p != NULL) {
+        if (!OSSL_PARAM_get_octet_string(
+                p, &poqs_sigctx->context_string, 0,
+                &(poqs_sigctx->context_string_length))) {
```

Comment:
```suggestion
                &poqs_sigctx->context_string_length)) {
```
The parens here aren't necessary, which isn't a problem _per se_, but it's a little confusing when they're included here but not on the line above.

Reply:
Removed as part of the commit adding verify_with_ctx so this is then functionally complete and no-one can complain about the in-symmetry.